lametro/views.py

```diff
@@ -97,6 +96,8 @@ def get_context_data(self, **kwargs):
             .order_by('-latest_date')
 
         context['related_bills'] = related_bills
+        
+        context['events'] = self.get_object().actions_and_agendas
```

Comment:
There are several calls to `self.get_object()` in this method. What do you think about doing one at the beginning of the method, then using the result for the subsequent operations, that way we aren't hitting the database every time it's referenced?

Reply:
Another opportunity for clean up: `self.get_object().actions.all()` is called twice (and we can eliminate the first call, because you're replacing the definition of actions here).